src/async_fuse/fuse/fuse_reply.rs

```diff
@@ -24,7 +24,7 @@ use super::protocol::{
 #[cfg(feature = "abi-7-18")]
 use super::protocol::{FuseNotifyCode::FUSE_NOTIFY_DELETE, FuseNotifyDeleteOut};
 
-/// This trait describes a type that can be converted to Vec<IoVec<&[u8]>>
+/// This trait describes a type that can be converted to Vec<`IoVec`<&[u8]>>
```

Comment:
Which clippy lint did this violate? Shall we say `Vec<IoVec<&[u8]>>` here?

Reply:
Clippy did suggest me to do it.

Reply:
Sorry, I mean, add the backticks (\`) outside the whole `Vec<IoVec<&[u8]>>`, as it is a complete type.

Reply:
I think current way is better. Cause `IoVec` is defined in this mod.